operator/Dockerfile.skaffold

```diff
@@ -46,7 +46,7 @@ ADD pkg $PROJECTPATH/pkg
 
 RUN make build
 
-# COPY ./preflight /root/.krew/bin/kubectl-preflight
+COPY ./preflight /root/.krew/bin/kubectl-preflight
```

Comment:
why this? I feel like either it should be for both preflight and support-bundle or neither